peps/pep-0725.rst

```diff
@@ -980,69 +1000,35 @@ Reusing the ``Requires-External`` field in Core Metadata
-``1.2.3a`` may indicate a release subsequent to ``1.2.3`` rather than an alpha version.
-To handle such cases correctly, it would be necessary to allow arbitrary
+There are cases, in particular when dealing with pre-releases, where PEP 440
+semantics for version comparisons don't quite work. For example, ``1.2.3a`` may
+indicate a release subsequent to ``1.2.3`` rather than an alpha version. To
+handle such cases correctly, it would be necessary to allow arbitrary
 versioning schemes. The authors of this PEP consider the added value of
-allowing that is not justified by the additional complexity. If desired,
-a package author can use either a code comment or the ``qualifier`` field of a
+allowing that is not justified by the additional complexity. If desired, a
+package author can use either a code comment or the ``qualifier`` field of a
 DepURL (see the Versioning section under Rationale) to capture this level of
 detail.
 
 Open Issues
 ===========
 
-Versioning of virtual dependencies
------------------------------------
-
-While virtual dependencies can be versioned with the same syntax, it must be better
-specified however what the version scheme is, because this is not as clear for
-virtual types as it is for PURL types (e.g., there can be multiple implementations,
-and virtual interfaces may not be unambiguously versioned). E.g.:
-
-- OpenMP: has regular ``MAJOR.MINOR`` versions of its standard, so would look
-  like ``>=4.5``.
-- BLAS/LAPACK: should use the versioning used by `Reference LAPACK`_, which
-  defines what the standard APIs are. Uses ``MAJOR.MINOR.MICRO``, so would look
-  like ``>=3.10.0``.
-- Compilers: these implement language standards. For C, C++ and Fortran these
-  are versioned by year. In order for versions to sort correctly, we choose to
-  use the full year (four digits). So "at least C99" would be ``>=1999``, and
-  selecting C++14 or Fortran 77 would be ``==2014`` or ``==1977`` respectively.
-  Other languages may use different versioning schemes. These should be
-  described somewhere before they are used in ``pyproject.toml``.
-
-A logistical challenge is where to describe the versioning - given that this
-will evolve over time, this PEP itself is not the right location for it.
-Instead, this PEP should point at that (to be created) location.
-
-Who defines canonical names and canonical package structure?
-------------------------------------------------------------
-
-Similarly to the logistics around versioning is the question about what names
-are allowed and where they are described. And then who is in control of that
-description and responsible for maintaining it. Our tentative answer is: there
-should be a central list for ``dep:generic`` and ``dep:virtual`` URLs,
-maintained as a PyPA project. See
-https://discuss.python.org/t/pep-725-specifying-external-dependencies-in-pyproject-toml/31888/62.
-TODO: once that list/project is prototyped, include it in the PEP and close
-this open issue.
```

Comment:
This is covered in https://github.com/rgommers/peps/blob/pep-name-mapping/peps/pep-9999.rst#central-registry, specifically at https://github.com/rgommers/peps/blob/4c5fc6a734210c457b349d66b80f92c5ca407416/peps/pep-9999.rst?plain=1#L198-L200